first-steps.mdx

```diff
@@ -1,137 +1,130 @@
-    alt="enter a test user for log-in tesst screen, screenshot 10/2023"
+    src="/images/setup/setup-5-login-credentials.png"
+    alt="AI agent asks for login test credentials, screenshot 07/2024"
   />
 </Frame>
 
-## 4. Create an account
+## 5. Discover more test cases
 
-Now we need to sign you up. Please, provide an email, so we can get in touch.
+Here comes the really cool part. You can let the AI agent to through your app and discover more tests. Click on `discover`.
 
-<Frame caption="Fourth page of the setup flow - create account, screenshot 08/2023">
+<Frame caption="Clicking this button will let our AI agent discover test cases, screenshot 07/2024">
   <img
-    src="/images/setup/setup-4-create-account.png"
-    alt="create account, screenshot 08/2023"
+    src="/images/setup/setup-6-discover.png"
+    alt="AI discovery button, screenshot 07/2024"
   />
 </Frame>
 
-## 5. Discover your test cases
+When it's done, the AI agent will let you know. You will find your tests on the `test cases` page.
+
+<Frame caption="AI agent discovered test cases and you will find them in the test cases view, screenshot 07/2024">
+  <img
+    src="/images/setup/setup-7-discovery-done.png"
+    alt="AI discovery has finished, screenshot 07/2024"
+  />
+</Frame>
+
+## 6. View your first AI auto-generated test cases
+
+You should have your **first active test cases** generated. The AI agent has discovered the test cases, it auto-generated the test steps and has run them to validate whether they work.
+
+They are `on` which means they will run once you trigger a **test report**.
+
+<Frame caption="First active test cases discovered and auto-generated - screenshot 07/2024">
+  <img
+    src="/images/setup/setup-8-first-tests.png"
+    alt="First active test cases AI discovered and AI auto-generated - screenshot 07/2024 "
+  />
+</Frame>
 
-Drafts of the test cases selected during sign-up can be found in the drafts section. Meanwhile, Octomind will discover test cases on your page according to how your website looks.
+If the agent stumbled over a test step, you might have to lend a hand to review and edit a test step.
 
-<Frame caption="Clicking this button will let our AI agent discover test cases, screenshot 02/2024">
+<Frame caption="Test cases that need review, discovered and auto-generated - screenshot 07/2024">
   <img
-    src="/images/discover.png"
-    width="300"
-    alt="discovery button, screenshot 02/2024"
+    src="/images/setup/setup-8-review-steps.png"
+    alt="Test cases that need review, AI discovered and AI auto-generated - screenshot 07/2024 "
   />
 </Frame>
 
-## 6. Approve your test cases
+## 7. Test your website for the first time
 
-Once our AI agent has finished discovery, you can approve or reject each individual draft to generate its steps.
+Once you have active test cases - they are **on**, you can **run** them. Create your first test report. Click on `run all` button and then `run in the app` and see if they found a bug.
```

Comment:
```suggestion
Once you have active test cases - they are **on**, you can **run** them. Create your first test report. Click on `run all` button and then `run in app` and see if they found a bug.
```
Good that you included that info here, test report and run all being connected is sth new users have to learn for conceptually at first I feel like